storage/client/file/src/main/java/com/azure/storage/file/FileStorageClient.java

```diff
@@ -17,53 +17,53 @@
 import java.util.Map;
 
 /**
- * This class provides a fileServiceAsyncClient that contains all the operations for interacting with a file account in Azure Storage.
- * Operations allowed by the fileServiceAsyncClient are creating, listing, and deleting shares and retrieving and updating properties
+ * This class provides a fileStorageAsyncClient that contains all the operations for interacting with a file account in Azure Storage.
+ * Operations allowed by the fileStorageAsyncClient are creating, listing, and deleting shares and retrieving and updating properties
  * of the account.
  *
  * <p><strong>Instantiating a Synchronous File Service Client</strong></p>
  *
- * {@codesnippet com.azure.storage.file.fileServiceClient.instantiation}
+ * {@codesnippet com.azure.storage.file.fileStorageClient.instantiation}
  *
- * <p>View {@link FileServiceClientBuilder this} for additional ways to construct the fileServiceAsyncClient.</p>
+ * <p>View {@link FileStorageClientBuilder this} for additional ways to construct the fileStorageAsyncClient.</p>
```

Comment:
The should be FileStorageClient, not fileStorageAsyncClient.

Reply:
Done.